migrate_pim/migrate_to_saas.rst

```diff
@@ -0,0 +1,139 @@
+Migrate to Serenity (SaaS)
```

Comment:
To our customers, Serenity and SaaS are the same thing. @betty-godier is checking how and when we will begin phasing out the Serenity naming conventions, but for now, I have included both "Serenity" and "SaaS" so that we don't confuse customers.

We can revise this in the future when we begin to phase out the Serenity naming convention across all docs.